### DISCUSSION TITLE
Ubuntu下用.deb安装包安装后如何启动

### DISCUSSION THREAD

**Discussion by RYDlen** [SlopeCraft/SlopeCraft#98] [Q&A｜问答] [answered]
Ubuntu22.04，安装SlopeCraft-5.2.0-Linux.deb后应用程序并无显示，终端输入slopecraft也没反应，是不是要去安装位置打开...

**Comment by ToKiNoBug** [comment]
> Ubuntu22.04，安装SlopeCraft-5.2.0-Linux.deb后应用程序并无显示，终端输入slopecraft也没反应，是不是要去安装位置打开...

没反应是指找不到应用程序，还是程序启动了，但不弹出窗口也不输出任何信息？

**Comment by ToKiNoBug** [comment, marked as answer]
应该输入的是SlopeCraft，Linux是严格区分大小写的

另外我确实没针对ubuntu专门弄更方便的图标，因为我现在用的不是ubuntu。不过我可以通过虚拟机来调试
